README.md

```diff
@@ -100,7 +100,7 @@ Options:
 -i --increment [<level>]
         Increment a version by the specified level.  Level can
         be one of: major, minor, patch, premajor, preminor,
-        prepatch, or prerelease.  Default level is 'patch'.
+        prepatch, prerelease, or release.  Default level is 'patch'.
```

Comment:
`inc()` is also documented under `#Functions` below and this should be added there.